lib/rspec/parameterized.rb

```diff
@@ -38,7 +38,17 @@ def initialize(arg_names, table_format, &block)
       #     end
       #
       def where(*args, &b)
-        set_parameters(args, false, &b)
+
+        if args.size == 1 && (params = args[0]).instance_of?(Hash)
```

Comment:
`(params = args[0]).instance_of?(Hash)` is not reader friendly.
`params` assignment should be in `if` block